### PR TITLE
Weizhuoz/enabling training part

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**

### CODE REVIEW

File: scripts/modelbench/launch.sh [hunk 1]
```diff
@@ -56,7 +56,9 @@ docker cp /home/ubuntu/docker/inductor_train.sh $USER:/workspace/pytorch
 docker cp /home/ubuntu/docker/version_collect.sh $USER:/workspace/pytorch
 
 if [ $TEST_MODE == "inference" ]; then
-    docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE inductor_log $DYNAMO_BENCH $WRAPPER $HF_TOKEN"
+    docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE inductor_log $DYNAMO_BENCH $WRAPPER $HF_TOKEN inference"
+elif [ $TEST_MODE == "training full scope" ]; then
```
Comment (chuanqi129) [inline scripts/modelbench/launch.sh:60]: suggest the `TEST_MODE` of full scope training  use `training_full`

File: scripts/modelbench/launch.sh [hunk 2]
```diff
@@ -56,7 +56,9 @@ docker cp /home/ubuntu/docker/inductor_train.sh $USER:/workspace/pytorch
...
+    docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE inductor_log $DYNAMO_BENCH $WRAPPER $HF_TOKEN training"
```
Comment (chuanqi129) [inline scripts/modelbench/launch.sh:61]: suggest to hardcode `THREADS` as `multiple` in training test

File: scripts/modelbench/launch.sh (hunk 1 above)
Comment (WeizhuoZhang-intel) [reply scripts/modelbench/launch.sh]: Resolved

File: scripts/modelbench/launch.sh (hunk 2 above)
Comment (WeizhuoZhang-intel) [reply scripts/modelbench/launch.sh]: Resolved

**Comment by chuanqi129** [comment]
@WeizhuoZhang-intel Please also consider the time cost evaluation and final report generate part in this PR.

**Comment by ESI-SYD** [comment]
Pls also add `training_full` test_mode option for `generate report` `artifacts` and `email` stage in groovy https://github.com/chuanqi129/inductor-tools/blob/weizhuoz/enabling_training_part/groovy/inductor_aws_benchmark.groovy#L457-L474
Also need to suit for taining , pls replace hard code `inference` in [report.py ](https://github.com/chuanqi129/inductor-tools/blob/weizhuoz/enabling_training_part/scripts/modelbench/report.py)

**Comment by chuanqi129** [comment]
@WeizhuoZhang-intel please rebase this PR to resolve the code conflict

**Pull request closed without merging**